meetings/conferences/serializers.py

```diff
@@ -1,22 +1,8 @@
-from rest_framework import serializers as ser
-from django_countries.fields import CountryField
+from rest_framework_json_api import serializers
 
 from conferences.models import Conference
 
-class ConferenceSerializer(ser.ModelSerializer):
-    country = CountryField() #get country_dict working later
```

Comment:
- required: id, Title, description (?)
- The country package seems to be serializing quite nicely without specific serialization, are you getting something different @Tav0 ?
- What metadata?